.Rbuildignore

```diff
@@ -1 +0,0 @@
-^~/\.rladies_meetup_token\.rds$
```

Comment:
why this change btw?

Reply:
website does not handle any meetup downloads, its in its own repo now. 
https://github.com/rladies/meetup_archive